tests/baselines/reference/bundlerImportTsExtensions(allowimportingtsextensions=true,noemit=false).errors.txt

```diff
@@ -83,5 +83,5 @@ error TS6054: File '/project/e.txt' has an unsupported extension. The only suppo
     import {} from "./a.ts";
     import {} from "./a.d.ts";
                    ~~~~~~~~~~
-!!! error TS2846: A declaration file cannot be imported without 'import type'. Did you mean to import an implementation file './a' instead?
+!!! error TS2846: A declaration file cannot be imported without 'import type'. Did you mean to import an implementation file './a.ts' instead?

```

Comment:
This is the error message that changed based on my small addition in the checker. It was suggesting `./a.js` before, which is ok, but looks a bit weird when the user has explicitly opted into allowing `.ts` extensions.